portal-web/docroot/html/taglib/ui/user_group_search_container_results/page.jsp

```diff
@@ -25,44 +25,24 @@ SearchContainer userGroupSearchContainer = (SearchContainer)request.getAttribute
 if (Validator.isNotNull(searchTerms.getKeywords())) {
 	useIndexer = true;
 }
-
-Indexer<?> indexer = IndexerRegistryUtil.nullSafeGetIndexer(UserGroup.class);
 %>
 
 <liferay-ui:search-container id="<%= userGroupSearchContainer.getId(request, namespace) %>" searchContainer="<%= userGroupSearchContainer %>">
 	<liferay-ui:search-container-results>
-		<c:choose>
```

Comment:
Thanks for the explanation. It makes sense.

Other than the two comments I've made on tests, the pull looks good to me.